packages/manager/modules/dedicated-universe-components/src/directives/checkboxSwitch/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+
+import checkboxSwitchDirective from './checkboxSwitch';
+import './checkboxSwitch.css';
+import './checkboxSwitch-2.css';
+
+const moduleName = 'ducCheckboxSwitch';
+
+angular
+  .module(moduleName, [])
+  .directive('checkboxSwitch', checkboxSwitchDirective);
```

Comment:
If you keep it, it should be named 
```suggestion
  .directive('ducCheckboxSwitch', checkboxSwitchDirective);
```

and usages should be updated to reflect it.

Reply:
I have removed it and prefixed with "duc" for all other providers